README.md

```diff
@@ -1,5 +1,62 @@
 # BC Parks Foundation
 
+BC Parks Foundation is a Mobile App build with React Native Technology that can be deployed in both ios and Android (with some ajustments) platforms.
+
+## <img src="./client/js/assets/images/discoverWhite.png" >
+
+## Some Screenshots from Mobile App:
+
+## Description
+
+BC Parks Foundation is a Mobile App to connect people to Outdoor hosted events or selfguided tours, where is possible to buy your ticket, save your favorite events/tours or get in touch with one the hosts.
```

Comment:
The description should be the first section of your README. Please move it.

Reply:
Done